Resources/views/DataCollector/arangodb.html.twig

```diff
@@ -12,7 +12,7 @@
         </div>
         <div class="sf-toolbar-info-piece">
             <b>Request time</b>
-            <span>{{ '%0.2f'|format(collector.totaltime) }} ms</span>
+            <span>{{ '%0.2f'|format(collector.totaltime * 1000) }} ms</span>
```

Comment:
Don't really have a setup right now to test this but this is looking wrong. The unit should be s instead of ms now? Maybe I am mistaken

Reply:
Nope. The `mircotime(true)` function tracks time in seconds, not in milliseconds. E.g if a call takes 6 milliseconds, you get 0.006, not 6.0 as result. So if you want to show the duration in milliseconds, you need to multiply by 1000.